recipes/umis/meta.yaml

```diff
@@ -37,9 +36,15 @@ requirements:
 test:
   commands:
     # click requires a unicode locale
-    - LANG=C.UTF-8 umis --help
+    - LANG=en_US.UTF-8 umis --help
```

Comment:
Maybe exporting `LANG=en_US.UTF-8` is not even needed for the extended container.

Reply:
Yes, I don't think it is needed.